src/vs/workbench/parts/terminal/electron-browser/terminalLinkHandler.ts

```diff
@@ -188,12 +238,65 @@ export class TerminalLinkHandler {
 			return TPromise.as(void 0);
 		}
 
+		const linkUrl = this.extractLinkUrl(link);
 		// Open an editor if the path exists
-		return pfs.fileExists(link).then(isFile => {
+		return pfs.fileExists(linkUrl).then(isFile => {
 			if (!isFile) {
 				return null;
 			}
 			return link;
 		});
 	}
+
+	/**
+	 * Returns line and column number of URl if that is present.
+	 *
+	 * @param link Url link which may contain line and column number.
+	 */
+	public extractLineColumnInfo(link: string): LineColumnInfo {
+		const matches: string[] = this._localLinkRegex.exec(link);
+
+		let lineColumnClauseLength = this._lineAndColumnClauses.length;
+		let lineColumnInfo: LineColumnInfo = {};
+
+		let lineAndColumnMatchIndex = this._winLineAndColumnMatchIndex;
+
+		if (platform.Platform.Windows !== this._platform) {
+			lineAndColumnMatchIndex = this._unixLineAndColumnMatchIndex;
+		}
+
+		for (let i = 0; i < lineColumnClauseLength; i++) {
+			let lineMatchIndex = lineAndColumnMatchIndex + 6 * i;
```

Comment:
while adding line and column regex, i have appended all regex clauses and each clause have 6 groups hence, after performing regex line and column information from each clause will be present at distance of 6. I think it would be better to have this value as constant up at top.